.github/workflows/python.yml

```
@@ -0,0 +1,27 @@
+# Python unit tests
+name: Python
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        python-version: [3.7]
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v2.1.1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          pip-upgrade
+          pip install -r requirements.txt
+          pip install -r requitements-dev.txt
+      - name: black
+        run: black flask_appbuilder
```

```suggestion
        run: black -l 90 flask_appbuilder
```
To be consisten tiwth flake8. See: https://github.com/dpgaspar/Flask-AppBuilder/blob/master/.flake8#L3